# Configuration file for the data collection of skewering
# 

data:
  numTrials: 5
  foods: ["strawberry"]
  tiltAngles: [0, 0, 0]
  tiltModes: [0, 0, 0]
  directions: [0, 45, 90, 135, 180, 225, 270, 315] #, 90, 135, 180, 225, 270, 315]
  angleNames: ["0", "45", "90", "135", "180", "225", "270", "315"] #, "S", "SE", "E", "NE", "N", "NW"]

# Feeding demo movement parameters
feedingDemo:
  heightAbovePlate: 0.13
  heightAboveFood: 0.11
  heightIntoFood: 0.02           # the real robot plans *below* the food. This parameter specifies how far
  heightOutOfPlate: 0.01
  distanceToPerson: 0.25
  distanceFromPerson: 0.15
  distBeforePush: 0.05
  distAfterPush: 0.05
  prePushOffset: -0.04
  distToPush: 0.20
  pushVelLimit: 0.2
  waitMillisecsAtFood: 500     # short pause after skewering the food
  waitMillisecsAtPerson: 5000   # long pause so person can take a bite

numTrials: 100

# Parameters for the F/T sensor and the MoveUntilTouchController
ftSensor:
  thresholds:
    standard:       # used when no other thresholds are specified
      force: 10
      torque: 2
    grabFood:       # active when going down to grab the food
      force: 15
      torque: 2
    afterGrabFood:  # active when going up after food has been grabbed
      force: 25 #3
      torque: 2
    pushFood:
      force:  4
      torque: 2